nemo/collections/asr/parts/submodules/subsampling.py

```diff
@@ -64,7 +64,7 @@ class ConvSubsampling(torch.nn.Module):
     VGGNet Subsampling: Transformer-transducer: end-to-end speech recognition with self-attention (https://arxiv.org/pdf/1910.12977.pdf)
     Striding Subsampling: "Speech-Transformer: A No-Recurrence Sequence-to-Sequence Model for Speech Recognition" by Linhao Dong et al. (https://ieeexplore.ieee.org/document/8462506)
     Args:
-        subsampling (str): The subsampling technique from {"vggnet", "striding"}
+        subsampling (str): The subsampling technique from {"vggnet", "striding", "dw-striding"}
```

Comment:
Stacking and stacking_norm are missing.

Reply:
`ConvSubsampling` does not support `stacking` and `stacking_norm`, so I did not add them.